htdocs/install/mysql/data/llx_accounting_abc.sql

```diff
@@ -129,3 +129,6 @@ INSERT INTO llx_accounting_system (fk_country, pcg_version, label, active) VALUE
 
 -- Description of chart of account DZ NSCF
 INSERT INTO llx_accounting_system (fk_country, pcg_version, label, active) VALUES (  13, 'NSCF', 'Nouveau système comptable financier', 1);
+
+-- Description of chart of account NL STANDAARD
+INSERT INTO llx_accounting_system (fk_country, pcg_version, label, active) VALUES (  17, 'STANDAARD-NL', 'Rekeningschema Nederland', 1);
```

Comment:
A previous PR already add this line
INSERT INTO llx_accounting_system (fk_country, pcg_version, label, active) VALUES ( 17, 'NL-VERKORT', 'Verkort rekeningschema', 1);
Is this new one requiered ?

Reply:
Hi Eldy
I am working on an better chart of accounts for the Dutch users. 
Still have some questions about it, so it can take a while.